audit/gather/gather.go

```diff
@@ -525,19 +527,67 @@ func (g *gather) captureServerProfiles(serverInfoMap map[string]*server.ServerIn
 }
 
 // Capture configured endpoints for each known server
-func (g *gather) captureServerEndpoints(serverInfoMap map[string]*server.ServerInfo) error {
+func (g *gather) captureServerEndpoints(serverInfoMap map[string]*server.ServerInfo, detail bool) error {
 	if g.aw == nil {
 		return fmt.Errorf("no archive writer supplied")
 	}
 
 	g.log.Infof("Querying %d endpoints on %d known servers...", len(g.cfg.ServerEndpointConfigs), len(serverInfoMap))
 	capturedCount := 0
+
+	if detail {
+		g.log.Infof("Collecting detailed server information. You can disable this feature by passing the --no-details flag")
```

Comment:
shouldn't mention the flag here as this is a library not the cli yet

Reply:
Gotcha, removed it.